### PR TITLE
in some cases StartAndEnd is null RemoveNodesStartAndEndResolver

### PR DESCRIPTION
**Pull request by alphpkeemik**
On empty lines 
```
/**
* Comment
* 
* @Annotation...
*/
```
fatal error occured, $removedPhpDocNodeInfo was null

### CODE REVIEW

File: packages/BetterPhpDocParser/Printer/RemoveNodesStartAndEndResolver.php [hunk 1]
```diff
@@ -26,9 +26,14 @@ public function resolve(PhpDocNode $originalPhpDocNode, PhpDocNode $currentPhpDo
         $lastEndPosition = null;
 
         foreach ($removedChildNodes as $removedChildNode) {
-            /** @var StartAndEnd $removedPhpDocNodeInfo */
+            /** @var StartAndEnd|null $removedPhpDocNodeInfo */
             $removedPhpDocNodeInfo = $removedChildNode->getAttribute(PhpDocAttributeKey::START_AND_END);
 
+            // it's not there when comment block has empty row "\s\*\n"
+            if (is_null($removedPhpDocNodeInfo)) {
```
Comment (samsonasik) [inline packages/BetterPhpDocParser/Printer/RemoveNodesStartAndEndResolver.php]: You can check instanceof check:

```suggestion
            if (! $removedPhpDocNodeInfo instanceof StartAndEnd) {

**Comment by samsonasik** [comment]
Thank you @alphpkeemik

File: packages/BetterPhpDocParser/Printer/RemoveNodesStartAndEndResolver.php (hunk 1 above)
Comment (alphpkeemik) [reply packages/BetterPhpDocParser/Printer/RemoveNodesStartAndEndResolver.php]: That was how my first change looked, but phpstan did not like it
image

Comment (samsonasik) [reply packages/BetterPhpDocParser/Printer/RemoveNodesStartAndEndResolver.php]: probably cache

**Pull request merged**